src/Orleans/Statistics/RuntimeStatisticsGroup.cs

```diff
@@ -132,14 +133,18 @@ internal void Start()
             {
                 logger.Warn(ErrorCode.PerfCounterNotRegistered,
                     "CPU & Memory perf counters did not initialize correctly - try repairing Windows perf counter config on this machine with 'lodctr /r' command");
-                return;
```

Comment:
we used to not log ANY runtime stat, even those that do not depend of perf. counters at all if `countersAvailable` is false.